### ISSUE TITLE
Python Error

### ISSUE THREAD

**Issue by russelltadams** [se1exin/Vultr-Dynamic-DNS#4]
Sorry I am not a coder. Following the directions in the README I get this result.  I have the correct API key and root domain filled in, I have setup the name servers properly. I am using python 3.4 on CentOS7 on my Vultr instance. Any help appreciated. 

[root@russweb Vultr-Dynamic-DNS]# python3 update.py 
Traceback (most recent call last):
  File "update.py", line 55, in <module>
    with urllib.request.urlopen(list_request) as list_response:
  File "/usr/lib64/python3.4/urllib/request.py", line 161, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib64/python3.4/urllib/request.py", line 470, in open
    response = meth(req, response)
  File "/usr/lib64/python3.4/urllib/request.py", line 580, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib64/python3.4/urllib/request.py", line 508, in error
    return self._call_chain(*args)
  File "/usr/lib64/python3.4/urllib/request.py", line 442, in _call_chain
    result = func(*args)
  File "/usr/lib64/python3.4/urllib/request.py", line 588, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
[root@russweb Vultr-Dynamic-DNS]#

**Comment by se1exin** [comment]
Hi @russelltadams - it appears you are getting a 403 HTTP error - which means the Vultr API is not authorising the request.

Second last line of your paste:
`urllib.error.HTTPError: HTTP Error 403: Forbidden` 

Have you enabled 'Allow All IPv4' in your Vultr Access Control settings? (Step 7 in the readme)

```
 - Enable API access and generate an API Key
   - Underneath 'Access Control' click 'Allow All IPv4' - this is essential as your IP is obviously undetermined (hence why you are using the script..)
```

**Comment by russelltadams** [comment]
So after some troubleshooting it turns out of if you have IPv6 enabled on your instance that the API service defaults to accepting your call on IPv6, or perhaps put another way, my host was sending the call via IPv6 by default. I had already clicked "Allow All IPv4", but I decided to regenerate a new API remove the rule and then re-add it, but I got the same result.  Then when I clicked "Allow All IPv6" it started working. I was able to remove the IPv4 rules and it still works. 

So in short, in this case, since IPv6 is enabled in the instance, I needed to "Allow All IPv6".  My OS seems to be preferring IPv6 by default, the call gets made over IPv6, and the IPv4 rule then of course doesn't work. 

Perhaps that can be added as a note in the readme, if your host prefers IPv6, then click "Allow All IPv6" instead. Seems like kind of a no brainer now, but of course I had forgotten I had enabled IPv6! 

Thanks for the script, working great.

**Comment by se1exin** [comment]
Thanks @russelltadams for figuring that out and for sharing the details in PR #5 !
